experiment/manual-trigger/manual-trigger.go

```diff
@@ -178,9 +178,9 @@ func main() {
 		MaxConcurrency: 1,
 	}
 
-	if err = jc.BuildFromSpec(&spec, o.jobName); err != nil {
+	if err = jc.BuildFromSpec(&spec, "0", o.jobName); err != nil {
```

Comment:
This is a bad API for the manual trigger but it's not super important to tackle right now.